<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortControlWidget</class>
 <widget class="QWidget" name="PortControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QComboBox" name="serialPortWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>73</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>73</width>
       <height>27</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="connectWidget">
     <property name="minimumSize">
      <size>
       <width>151</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>151</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>No COM Port Available</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="minimumSize">
      <size>
       <width>84</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>84</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
